packages/tools/src/groupFilesByType.ts

```diff
@@ -32,5 +21,5 @@ mime.define({
  * the returned object will be: {font: ['fonts/a.ttf'], image: ['images/b.jpg']}
  */
 export default function groupFilesByType(assets: Array<string>) {
-  return groupBy(assets, type => mime.lookup(type).split('/')[0]);
+  return groupBy(assets, type => (mime.getType(type) || '').split('/')[0]);
```

Comment:
When this can return `undefined`?

Reply:
When you try ot get mime type which doesn't exist.

Reply:
Was it always like that, even with the previous version? Trying to get an idea whether we had a hidden `cannot read property split of undefined` error or the method changed.

Reply:
Probably yes, because `mime` lib didn't have any flow typings. When upgraded `mime` lib and attaching typescript types, we caught this error.